### PR TITLE
Add: periodic job to remove ghost locks. (#5087)

### PR DESCRIPTION
**Pull request by mikewilli**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [x] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

## Description
Adds a scheduled job to periodically clear out locks on worker queues.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -34,6 +34,7 @@ SQLAlchemy-Utils==0.34.2
 sqlparse==0.3.0
 statsd==3.3.0
 gunicorn==19.9.0
+greenlet==0.4.16
```
Comment (edugfilho) [inline requirements.txt:37]: What's this one for?

File: redash/tasks/schedule.py [hunk 2]
```diff
@@ -62,6 +63,11 @@ def schedule(kwargs):
 def periodic_job_definitions():
     jobs = [
         {"func": refresh_queries, "interval": 30, "result_ttl": 600},
+        {
+            "func": remove_ghost_locks,
+            "interval": timedelta(minutes=1),
+            "result_ttl": 600,
```
Comment (edugfilho) [inline redash/tasks/schedule.py:69]: just curious: does it mean this will run every minute for a maximum time of 10 minutes?

**Review by edugfilho — APPROVED** [review]
one minor comment and a question

File: requirements.txt (hunk 1 above)
Comment (mikewilli) [reply requirements.txt:37]: Yea this probably should've been its own PR, but it's now needed to be specified explicitly due to a conflict in dependencies.

File: redash/tasks/schedule.py (hunk 2 above)
Comment (mikewilli) [reply redash/tasks/schedule.py:69]: Yep, after `result_ttl` (which defaults to `interval * 2` in Redash) seconds, the job and any results will be cleaned up.

**Pull request merged**